extensions/jython/tests/conf/tests.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+ 
+<suite name="Jython extension">
+  <test name="tests">
+    <groups>
+      <run>
+        <exclude name="broken" />
```

Comment:
What does this do exactly ?

Reply:
Yep, that just lets you skip tests that are known to fail for some reason.

Reply:
how does it know they have failed before ? it keeps some history or xml in a folder , or ?

Reply:
it's something you add manually, when you are fed up by one particular test that fails because of something outside your control

Reply:
ah k